conductor-clients/java/conductor-java-sdk/conductor-client-spring/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,2 @@
+com.netflix.conductor.client.spring.ConductorClientAutoConfiguration
```

Comment:
This is necessary to ensure everything works without requiring the respective packages to be included in the component scan.